### PR TITLE
Polkadot nominator and validator guides

### LINKED ISSUE #451: Add Nominating and Validating on Polkadot pages
Equivalent to our ["Be a Nominator (Kusama)"](https://wiki.polkadot.network/docs/en/mirror-maintain-guides-how-to-nominate-kusama) and ["Be a Validator (Kusama)"](https://wiki.polkadot.network/docs/en/mirror-maintain-guides-how-to-validate-kusama) pages

### LINKED ISSUE #464: Nominator page uses obsolete naming and URL
https://wiki.polkadot.network/docs/en/maintain-nominator page uses obsolete naming and url https://polkadot.js.org/apps/#/staking/returns

### PR DESCRIPTION
**Pull request by lsaether**
closes #451 
closes #464

### CODE REVIEW

File: docs/maintain-guides-how-to-nominate-kusama.md [hunk 1]
```diff
@@ -27,20 +27,34 @@ detailed on the [account generation][] page.
 
 ### Step 1: Bond your tokens
 
-On the [Polkadot UI](https://polkadot.js.org/apps) navigate to the "Staking" tab.
+On the [Polkadot UI](https://polkadot.js.org/apps) navigate to the "Staking" tab. The "Staking
+Overview" subsection will show you all the active validators and their information - their
+identities, the amount of DOTs that are staking for them, amount that is their own provided stake,
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
identities, the amount of KSM that are staking for them, amount that is their own provided stake,
```

This is for the Kusama section, right? (or at least that's what the filename implies.

File: docs/maintain-guides-how-to-nominate-kusama.md [hunk 2]
```diff
@@ -27,20 +27,34 @@ detailed on the [account generation][] page.
...
 
-The "Staking Overview" subsection will show you all the active validators and their information -
-points (reliability), earnings, identities, etc. The "Waiting" subsection lists all pending
-validators that need more nominations to enter the active validator set. The "Returns" screen will
-help you estimate your earnings and this is where it's good to start picking favorites. The "Account
-Actions" subsection allows you to stake and nominate and the "Validator Stats" screen will show you
-some interesting in-depth graphs about a selected validator.
+The "Waiting" subsection ([link](https://polkadot.js.org/apps/#/staking/waiting)) lists all pending
+validators that are awaiting more nominations to enter the active validator set. Validators will
+stay in the waiting queue until they have enough DOTs backing them (as allocated through the
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
stay in the waiting queue until they have enough KSM backing them (as allocated through the
```

File: docs/maintain-guides-how-to-nominate-kusama.md [hunk 3]
```diff
@@ -27,20 +27,34 @@ detailed on the [account generation][] page.
...
+Phragmen election mechanism). It is possible validator can remain in the queue for a very long time
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
[Phragmen election mechanism](https://wiki.polkadot.network/docs/en/learn-phragmen)). It is possible validator can remain in the queue for a very long time
```

File: docs/maintain-guides-how-to-nominate-kusama.md [hunk 4]
```diff
@@ -27,20 +27,34 @@ detailed on the [account generation][] page.
...
+if they never get enough backing.
 
-Pick "Account Actions", then click the blue "New Stake" button.
+The "Targets" subsection ([link](https://polkadot.js.org/apps/#/staking/targets)) will help you
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-kusama.md:44]: Would list them in order of:

1. Account Actions
2. Payouts
3. Targets

As this is the order they appear in on the UI

File: docs/maintain-guides-how-to-nominate-kusama.md [hunk 5]
```diff
@@ -27,20 +27,34 @@ detailed on the [account generation][] page.
...
+estimate your earnings and this is where it's good to start picking favorites.
+
+The "Account actions" subsection ([link](https://polkadot.js.org/apps/#/staking/actions)) allows you
+to stake and nominate.
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-kusama.md]:
```suggestion
to stake and nominate. Note, this tab will only show up when you have existing accounts in the [accounts](https://polkadot.js.org/apps/#/accounts) page.
```

File: docs/maintain-guides-how-to-nominate-kusama.md [hunk 6]
```diff
@@ -27,20 +27,34 @@ detailed on the [account generation][] page.
...
+
+The "Validator Stats" screen will show you some interesting in-depth graphs about a selected
+validator.
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-kusama.md]: Would reword this for more detail:

The "Validator Stat's subsection ([link](https://polkadot.js.org/apps/#/staking/query)) allows you to query a validator's stash address and see historical charts on era points, elected stake, rewards, and slashes.

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+> The following information applies to the Polkadot network which is currently not yet launched. If
+> you want to get started today please consider using this guide on the Westend test network, or
+> checking out the [Kusama guide](maintain-guides-how-to-nominate-kusama) instead.
+
+Nominators are one type of participant in the staking subsystem of Polkadot. They are responsible
+for appointing their stake to the validators who are the second type of participant. By appointing
+their stake, they are able to elect the active set of validators and share in the rewards that are
+paid out.
+
+While the [validators][] are active participants in the network that take part in the block
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md:17]: Link to validator page

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+production and finality mechanisms, nominators take a more passive role that can compared to the
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
production and finality mechanisms, nominators take a more passive role with a
```

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+"set-it-and-forget-it" approach. Being a nominator does not require running a node of your own or
+worrying about online uptime. However, a good nominator performs diligence on the validators that it
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
worrying about online uptime. However, a good nominator performs diligence on the validators that they
```

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+elects. When looking for validators to nominate, a nominator should pay attention to its own reward
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
elect. When looking for validators to nominate, a nominator should pay attention to their own reward
```

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+percentage for nominating a specific validator - as well as the risk that it bares of being slashed
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
percentage for nominating a specific validator - as well as the risk that they bare of being slashed
```

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+if the validator gets slashed.
+
+# Setting up Stash and Controller keys
+
+Nominators are recommended to set up two separate stash and controller accounts. Explanation and
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
Nominators are recommended to set up separate stash and controller accounts. Explanation and
```
The two here kind of sounds like they need two stash and two controller accounts

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 13]
```diff
@@ -0,0 +1,181 @@
...
+reasoning for generating distinct accounts for this purpose is elaborated in the [keys][] section of
+the Wiki.
+
+You can generate your stash and controller account via any of the recommended methods that are
+detailed on the [account generation][] page.
+
+# Using Polkadot UI
+
+### Step 1: Bond your tokens
+
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md:37]: Would maybe add a note here that this will be applicable when NPoS change happens. There's a disclaimer at the beginning, but also adding a brief one here wouldn't hurt.

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 14]
```diff
@@ -0,0 +1,181 @@
...
+if they never get enough backing.
+
+The "Targets" subsection ([link](https://polkadot.js.org/apps/#/staking/targets)) will help you
+estimate your earnings and this is where it's good to start picking favorites.
+
+The "Account actions" subsection ([link](https://polkadot.js.org/apps/#/staking/actions)) allows you
+to stake and nominate.
+
+The "Validator Stats" screen will show you some interesting in-depth graphs about a selected
+validator.
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]: Would copy the changes from above into here as well.

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 15]
```diff
@@ -0,0 +1,181 @@
...
+You are now bonded. Being bonded means your tokens are locked and could be
+[slashed](learn-staking#slashing) if the validators you nominate misbehave. All bonded funds can now
+be distributed to up to 16 validators. Be careful about the validators you choose since you will be
+slashed if your validator commits an offence.
+
+Click on "Nominate" on an account you've bonded and you will be presented with another popup asking
+you to select up to 16 validators.
+
+![Nominating validators](/img/NPoS/nominate.png)
+
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md:98]: Would add a note here that although you can nominate all 16, the stake will be dispersed via phragmen, and you may not actually actively nominate all the ones you choose.

File: website/sidebars.json [hunk 16]
```diff
@@ -145,7 +145,7 @@
         "label": "Nominator",
         "ids": [
           "maintain-nominator",
-          "maintain-guides-how-to-nominate-kusama",
+          "maintain-guides-how-to-nominate-polkadot",
```
Comment (wpank) [inline website/sidebars.json:148]: These should be added in addition, not changing this line. (There should be both pages on kusama and polkadot)

File: website/sidebars.json [hunk 17]
```diff
@@ -154,7 +154,7 @@
         "label": "Validator",
         "ids": [
           "maintain-validator",
-          "maintain-guides-how-to-validate-kusama",
+          "maintain-guides-how-to-validate-polkadot",
```
Comment (wpank) [inline website/sidebars.json:157]: Same as above, this should be an addition, not a change.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 18]
```diff
@@ -0,0 +1,392 @@
+---
+id: maintain-guides-how-to-validate-kusama
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
id: maintain-guides-how-to-validate-polkadot
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 19]
```diff
@@ -0,0 +1,392 @@
...
+title: Run a Validator (Kusama)
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
title: Run a Validator (Polkadot)
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 20]
```diff
@@ -0,0 +1,392 @@
...
+that you can use by deploying yourself (video walkthrough is available
+[here](https://www.youtube.com/watch?v=tTn8P6t7JYc)). As you progress in your journey as a
+validator, you will likely want to use this repository as a _starting point_ for your own
+modifications and customizations.
+
+If you need help, please reach out on the
+[Polkadot Validator Longue](https://matrix.to/#/!NZrbtteFeqYKCUGQtr:matrix.parity.io?via=matrix.parity.io&via=matrix.org&via=web3.foundation)
+on Riot. The team and other validators are there to help answer questions and provide tips from
+experience. If you have a more significant proposal, you can write it on the
+[Web3 Foundation forum](https://forum.web3.foundation/).
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]: I heard word of deprecating the discourse forums - nothing really happens here. Is this worth mentioning?

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 21]
```diff
@@ -0,0 +1,392 @@
...
+> `git clone git@github.com:paritytech/polkadot.git`.
+
+```sh
+git clone https://github.com/paritytech/polkadot.git
+cd polkadot
+git tag | grep "$v\0\.7"
+git checkout v0.7.28
+./scripts/init.sh
+cargo build --release
+```
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]: This will change, but not really sure what it will look like in the meantime.....

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 22]
```diff
@@ -0,0 +1,392 @@
...
+directory. You may then take the generated `subkey` executable and transfer it to an air-gapped
+machine for extra security.
+
+```sh
+cargo install --force --git https://github.com/paritytech/substrate subkey
+```
+
+### Synchronize Chain Data
+
+> **Note:** Validators must sync their nodes in archive mode to avoid being slashed. If you've
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]: I think this can change a bit - validators won't be slashed, but their nodes just won't start (which can mean slashing).

**Note:** If a Validator syncs as a full node, they must sync their nodes in archive mode, otherwise the node will throw an error when on start.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 23]
```diff
@@ -0,0 +1,392 @@
...
+> already synced the chain, you must first remove the database with `polkadot purge-chain` and then
+> ensure that you run Polkadot with the `--pruning=archive` option.
+
+You can begin syncing your node by running the following command:
+
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md:181]: There should be something here on rocksdb vs paritydb - right now people should use rocksdb, but paritydb will be faster and eventually we will switch to that. If you use one vs the other it will require a resync.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 24]
```diff
@@ -0,0 +1,392 @@
...
+Depending on the size of the chain when you do this, this step may take anywhere from a few minutes
+to a few hours.
+
+If you are interested in determining how much longer you have to go, your server logs (printed to
+STDOUT from the `polkadot` process) will tell you the latest block your node has processed and
+verified. You can then compare that to the current highest block via
+[Telemetry](https://telemetry.polkadot.io/#list/Kusama) or the
+[PolkadotJS Block Explorer](https://polkadot.js.org/apps/#/explorer).
+
+> **Note:** If you do not already have KSM, this is as far as you will be able to go until the end
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
> **Note:** If you do not already have DOTs, this is as far as you will be able to go until the end
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 25]
```diff
@@ -0,0 +1,392 @@
...
+> of the soft launch period. You can still run a node, but you will need to have a minimal amount of
+> KSM to continue, as balance transfers are disabled during the soft launch. Please keep in mind
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
> DOTs to continue, as balance transfers are disabled during the soft launch. Please keep in mind
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 26]
```diff
@@ -0,0 +1,392 @@
...
+> that even for those with KSM, they will only be indicating their _intent_ to validate; they will
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
> that even for those with DOTs, they will only be indicating their _intent_ to validate; they will
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 27]
```diff
@@ -0,0 +1,392 @@
...
+> also not be able to run a validator until the NPoS phase starts.
+
+## Bond KSM
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
## Bond DOTs
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 28]
```diff
@@ -0,0 +1,392 @@
...
+
+It is highly recommended that you make your controller and stash accounts be two separate accounts.
+For this, you will create two accounts and make sure each of them have at least enough funds to pay
+the fees for making transactions. Keep most of your funds in the stash account since it is meant to
+be the custodian of your staking funds.
+
+Make sure not to bond all your KSM balance since you will be unable to pay transaction fees from
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
Make sure not to bond all your DOT balance since you will be unable to pay transaction fees from
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 29]
```diff
@@ -0,0 +1,392 @@
...
+your bonded balance.
+
+It is now time to set up our validator. We will do the following:
+
+- Bond the KSM of the Stash account. These KSM will be put at stake for the security of the network
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
- Bond the DOTs of the Stash account. These DOTs will be put at stake for the security of the network
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 30]
```diff
@@ -0,0 +1,392 @@
...
+  and can be slashed.
+- Select the Controller. This is the account that will decide when to start or stop validating.
+
+First, go to the [Staking](https://polkadot.js.org/apps/#/staking/actions) section. Click on
+"Account Actions", and then the "New stake" button.
+
+![dashboard bonding](assets/guides/how-to-validate/polkadot-dashboard-bonding.jpg)
+
+- **Stash account** - Select your Stash account. In this example, we will bond 100 milliKSMs - make
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
- **Stash account** - Select your Stash account. In this example, we will bond 100 milliDOTs - make
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 31]
```diff
@@ -0,0 +1,392 @@
...
+  sure that your Stash account contains _at least_ this much. You can, of course, stake more than
+  this.
+- **Controller account** - Select the Controller account created earlier. This account will also
+  need a small amount of KSM in order to start and stop validating.
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
  need a small amount of DOTs in order to start and stop validating.
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 32]
```diff
@@ -0,0 +1,392 @@
...
+- **Value bonded** - How much KSM from the Stash account you want to bond/stake. Note that you do
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
- **Value bonded** - How much DOts from the Stash account you want to bond/stake. Note that you do
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 33]
```diff
@@ -0,0 +1,392 @@
...
+  not need to bond all of the KSM in that account. Also note that you can always bond _more_ KSM
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
  not need to bond all of the DOTs in that account. Also note that you can always bond _more_ DOTs
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 34]
```diff
@@ -0,0 +1,392 @@
...
+> to check for a specific key or
+> [hasSessionKeys](https://polkadot.js.org/api/substrate/rpc.html#hassessionkeys-sessionkeys-bytes-bool)
+> to check the full session key public key string.
+
+Once your node is fully synced, stop the process by pressing Ctrl-C. At your terminal prompt, you
+will now start running the node in validator mode with a flag allowing unsafe RPC calls, needed for
+some advanced operations.
+
+```sh
+./target/release/polkadot --validator --name "name on telemetry" --unsafe-rpc-expose
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
./target/release/polkadot --validator --name "name on telemetry"
```

You shouldn't actually need the --unsafe-rpc-expose flag anymore, the addition of this initially was a bug.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 35]
```diff
@@ -0,0 +1,392 @@
...
+```
+
+You can give your validator any name that you like, but note that others will be able to see it, and
+it will be included in the list of all servers using the same telemetry server. Since numerous
+people are using telemetry, it is recommended that you choose something likely to be unique. Note
+that the `--rpc-methods=Unsafe` flag is needed in order to set the session key as shown below. Note
+this flag only exposes RPC methods locally.
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]: Can remove the note on the unsafe rpc methods.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 36]
```diff
@@ -0,0 +1,392 @@
...
+
+```sh
+curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "author_rotateKeys", "params":[]}' http://localhost:9933
+```
+
+The output will have a hex-encoded "result" field. The result is the concatenation of the four
+public keys. Save this result for a later step.
+
+You can restart your node at this point, omitting the `--unsafe-rpc-expose` flag as it is no longer
+needed.
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md:316]: Remove mention of unsafe rpc.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 37]
```diff
@@ -0,0 +1,392 @@
...
+
+![staking-change-session](assets/guides/how-to-validate/set-session-key-1.jpg)
+![staking-session-result](assets/guides/how-to-validate/set-session-key-2.jpg)
+
+Submit this extrinsic and you are now ready to start validating.
+
+## Validate
+
+To verify that your node is live and synchronized, head to
+[Telemetry](https://telemetry.polkadot.io/#/Kusama) and find your node. Note that this will show all
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]: Link should be polkadot telemetry

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 38]
```diff
@@ -0,0 +1,392 @@
...
+nodes on the Kusama network, which is why it is important to select a unique name!
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
nodes on the Polkadot network, which is why it is important to select a unique name!
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 39]
```diff
@@ -0,0 +1,392 @@
...
+it will remain in the _waiting_ queue. If your validator is not selected to become part of the
+validator set, it will remain in the _waiting_ queue until it is. There is no need to re-start if
+you are not selected for the validator set in a particular era. However, it may be necessary to
+increase the number of KSMs staked or seek out nominators for your validator in order to join the
+validator set.
+
+**Congratulations!** If you have followed all of these steps, and been selected to be a part of the
+validator set, you are now running a Kusama validator! If you need help, reach out on the
+[Kusama forum](https://forum.kusama.network/) or in the
+[Kusama Validator chat](https://riot.im/app/#/room/#KusamaValidatorLounge:polkadot.builders).
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]: Replace with Polkadot links.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 40]
```diff
@@ -0,0 +1,392 @@
...
+The validator set is refreshed every era. In the next era, if there is a slot available and your
+node is selected to join the validator set, your node will become an active validator. Until then,
+it will remain in the _waiting_ queue. If your validator is not selected to become part of the
+validator set, it will remain in the _waiting_ queue until it is. There is no need to re-start if
+you are not selected for the validator set in a particular era. However, it may be necessary to
+increase the number of KSMs staked or seek out nominators for your validator in order to join the
+validator set.
+
+**Congratulations!** If you have followed all of these steps, and been selected to be a part of the
+validator set, you are now running a Kusama validator! If you need help, reach out on the
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
validator set, you are now running a Polkadot validator! If you need help, reach out on the
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 41]
```diff
@@ -0,0 +1,392 @@
...
+- [Contabo](https://contabo.com/)
+- [Scaleway](https://www.scaleway.com/)
+
+## Using Docker
+
+If you have Docker installed, you can use it to start your validator node without needing to build
+the binary. You can do this with a simple one line command:
+
+```sh
+$ docker run parity/polkadot:v0.7.28 --validator --name "name on telemetry"
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
$ docker run parity/polkadot:latest --validator --name "name on telemetry"
```

**Review by wpank — CHANGES_REQUESTED** [review]

File: docs/maintain-guides-how-to-nominate-kusama.md (hunk 4 above)
Comment (lsaether) [reply docs/maintain-guides-how-to-nominate-kusama.md:44]: good point

File: website/sidebars.json (hunk 16 above)
Comment (lsaether) [reply website/sidebars.json:148]: I think it makes sense that the Polkadot wiki only displays the sidebar for the Polkadot guide and the Kusama guide shows the Kusama ones.

File: docs/maintain-guides-how-to-validate-polkadot.md (hunk 20 above)
Comment (lsaether) [reply docs/maintain-guides-how-to-validate-polkadot.md]: It doesn't get much attention. Probably opening up an issue on one of the relevant repositories on GitHub gets more visibility.

File: docs/maintain-guides-how-to-nominate-polkadot.md [hunk 42]
```diff
@@ -0,0 +1,190 @@
...
+Nominators are one type of participant in the staking subsystem of Polkadot. They are responsible
+for appointing their stake to the validators who are the second type of participant. By appointing
+their stake, they are able to elect the active set of validators and share in the rewards that are
+paid out.
+
+While the [validators][] are active participants in the network that take part in the block
+production and finality mechanisms, nominators take a more passive role with a
+"set-it-and-forget-it" approach. Being a nominator does not require running a node of your own or
+worrying about online uptime. However, a good nominator performs diligence on the validators that
```
Comment (wpank) [inline docs/maintain-guides-how-to-nominate-polkadot.md]:
```suggestion
worrying about online uptime. However, a good nominator performs due diligence on the validators that
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 43]
```diff
@@ -0,0 +1,392 @@
...
+
+```sh
+sudo ntpq -p
+```
+
+### Building and Installing the `polkadot` Binary
+
+You will need to build the `polkadot` binary from the
+[paritytech/polkadot](https://github.com/paritytech/polkadot) repository on GitHub using the source
+code available in the **v0.7** branch.
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
code available in the **v0.8** branch.
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 44]
```diff
@@ -0,0 +1,392 @@
...
+
+You should generally use the latest **0.7.x** tag. At the time of writing, this was **0.7.28**, but
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
You should generally use the latest **0.8.x** tag. At the time of writing, this was **0.8.0**, but
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 45]
```diff
@@ -0,0 +1,392 @@
...
+you should review the output from the "git tag" command (`git tag | grep "$v\0\.7"`) to see a list
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
you should review the output from the "git tag" command (`git tag | grep "$v\0\.8"`) to see a list
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 46]
```diff
@@ -0,0 +1,392 @@
...
+of all the potential 0.7 releases. You should replace `v0.7.28` with the latest build (i.e., the
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
of all the potential 0.8 releases. You should replace `v0.8.0` with the latest build (i.e., the
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 47]
```diff
@@ -0,0 +1,392 @@
...
+highest number). You can also find the latest Kusama version on the
+[release](https://github.com/paritytech/polkadot/releases) tab.
+
+> Note: If you prefer to use SSH rather than HTTPS, you can replace the first line of the below with
+> `git clone git@github.com:paritytech/polkadot.git`.
+
+```sh
+git clone https://github.com/paritytech/polkadot.git
+cd polkadot
+git tag | grep "$v\0\.7"
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
git tag | grep "$v\0\.8"
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 48]
```diff
@@ -0,0 +1,392 @@
...
+git checkout v0.7.28
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
git checkout v0.8.0
```

File: docs/maintain-guides-how-to-validate-polkadot.md (hunk 22 above)
Comment (wpank) [reply docs/maintain-guides-how-to-validate-polkadot.md]: Actually you do not need to run in archive mode, you can run in unsafe pruning mode with the following flags:

`--unsafe-pruning --pruning <pruning number>`

Where pruning number can be something like 16384

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 49]
```diff
@@ -0,0 +1,392 @@
...
+[PolkadotJS Block Explorer](https://polkadot.js.org/apps/#/explorer).
+
+> **Note:** If you do not already have DOTs, this is as far as you will be able to go until the end
+> of the soft launch period. You can still run a node, but you will need to have a minimal amount of
+> DOTs to continue, as balance transfers are disabled during the soft launch. Please keep in mind
+> that even for those with DOTs, they will only be indicating their _intent_ to validate; they will
+> also not be able to run a validator until the NPoS phase starts.
+
+## Bond DOTs
+
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md:213]: Add a note here that until transfers are enabled, you _must_ use the same account if you did not get DOT injections done.

File: docs/maintain-guides-how-to-validate-polkadot.md (hunk 35 above)
Comment (wpank) [reply docs/maintain-guides-how-to-validate-polkadot.md]: With the V0.8.0 release, the necessity of this was removed.

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 50]
```diff
@@ -0,0 +1,392 @@
...
+![staking-change-session](assets/guides/how-to-validate/set-session-key-1.jpg)
+![staking-session-result](assets/guides/how-to-validate/set-session-key-2.jpg)
+
+Submit this extrinsic and you are now ready to start validating.
+
+## Validate
+
+To verify that your node is live and synchronized, head to
+[Telemetry](https://telemetry.polkadot.io/#/Kusama) and find your node. Note that this will show all
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
[Telemetry](https://telemetry.polkadot.io/#list/Polkadot%20CC1) and find your node. Note that this will show all
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 51]
```diff
@@ -0,0 +1,392 @@
...
+it will remain in the _waiting_ queue. If your validator is not selected to become part of the
+validator set, it will remain in the _waiting_ queue until it is. There is no need to re-start if
+you are not selected for the validator set in a particular era. However, it may be necessary to
+increase the number of KSMs staked or seek out nominators for your validator in order to join the
+validator set.
+
+**Congratulations!** If you have followed all of these steps, and been selected to be a part of the
+validator set, you are now running a Polkadot validator! If you need help, reach out on the
+[Kusama forum](https://forum.kusama.network/) or in the
+[Kusama Validator chat](https://riot.im/app/#/room/#KusamaValidatorLounge:polkadot.builders).
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
[Polkadot Validator chat](https://matrix.to/#/!NZrbtteFeqYKCUGQtr:matrix.parity.io?via=matrix.parity.io&via=matrix.org&via=web3.foundation).
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 52]
```diff
@@ -0,0 +1,392 @@
...
+node is selected to join the validator set, your node will become an active validator. Until then,
+it will remain in the _waiting_ queue. If your validator is not selected to become part of the
+validator set, it will remain in the _waiting_ queue until it is. There is no need to re-start if
+you are not selected for the validator set in a particular era. However, it may be necessary to
+increase the number of KSMs staked or seek out nominators for your validator in order to join the
+validator set.
+
+**Congratulations!** If you have followed all of these steps, and been selected to be a part of the
+validator set, you are now running a Polkadot validator! If you need help, reach out on the
+[Kusama forum](https://forum.kusama.network/) or in the
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
[Web3 Foundation forum](https://forum.web3.foundation/) or in the
```

File: docs/maintain-guides-how-to-validate-polkadot.md [hunk 53]
```diff
@@ -186,10 +185,14 @@ You can begin syncing your node by running the following command:
...
-**Note:** The `--pruning=archive` flag is implied by the `--validator` and `--sentry` flags, so it
-is only required explicitly if you start your node without one of these two options. If you do not
-set your pruning to archive node, even when not running in validator and sentry mode, you will need
-to re-sync your database when you switch.
+The `--pruning=archive` flag is implied by the `--validator` and `--sentry` flags, so it is only
+required explicitly if you start your node without one of these two options. If you do not set your
+pruning to archive node, even when not running in validator and sentry mode, you will need to
+re-sync your database when you switch.
+
+> **Note:** Validators should sync using the RocksDb backend by passing the `--databse RocksDb`
```
Comment (wpank) [inline docs/maintain-guides-how-to-validate-polkadot.md]:
```suggestion
> **Note:** Validators should sync using the RocksDb backend by passing the `--database RocksDb`
```

**Review by wpank — APPROVED** [review]
